version.sh

```diff
@@ -53,28 +53,29 @@ if [ -z ${VERSION} ]; then
         # head to grab the first one, since the list is sorted in descending
         # order, handling -rc tags correctly courtesy of
         # versionsort.suffix.
-        MAYBE_TAG=$(git \
-                        -c versionsort.suffix='-rc' \
-                        tag \
-                        --sort='-v:refname:short' \
-                        --format='%(refname:short)' \
-                        --no-contains=$(git merge-base HEAD main) | \
-                    grep -E -v 'v0.12|-ee' | \
-                    head -n 1
-                  )
+        MAYBE_TAG=$(
+            git \
+                -c versionsort.suffix='-rc' \
+                tag \
+                --sort='-v:refname:short' \
+                --format='%(refname:short)' \
+                --no-contains=$(git merge-base HEAD main) \
+                | grep -E -v 'v0.12|-ee' \
+                | head -n 1
+        )
     fi
 
     # Munge the tag into the form we want. Note: we always append a SHA hash,
     # even if we're on the commit with the tag. This is partially because I feel
     # like it will be more consistent and result in fewer surprises, but also it
     # might help indicate that this is a local version. Additionally, remove the
     # 'v' from the final version string.
-    echo -n "${MAYBE_TAG}+${SHA}" | tr -d 'v'
+    echo -n "${MAYBE_TAG}+${SHA}" | sed -e 's/^v//'
```

Comment:
This is bash, so I suppose I'll just throw this one out there too. ;)
```suggestion
    echo -n "${MAYBE_TAG#v}+${SHA}"
```
I do like using sed to make sure it's at the beginning rather than "anywhere at all" though.

Reply:
Huh, I learned a new shell expansion operator today! I checked some cases and it seems like it only matches at the front.